tests/src/JIT/Performance/CodeQuality/HWIntrinsic/X86/PacketTracer/Camera.cs

```diff
@@ -0,0 +1,30 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+//
+
```

Comment:
General nit: It would be useful to format all these documents according to the recommended conventions....

Things like one statement per line, braces on their own line, no stray newlines, no trailing whitespace, a newline after the top level using statements and before the first namespace, etc....

Reply:
I believe, VS, for the most part, should do this if you run the "Format Document" command

Reply:
Thanks! Will do.

Reply:
@fiigii, did this happen? It looks like we still have things like:
* Stray or missing newlines
* Unsorted usings
* Methods entirely on a single line
* etc